ckanext/geocat/utils/ogdch_map_utils.py

```diff
@@ -89,16 +89,26 @@ def map_to_ogdch_categories(geocat_categories):
 
 def map_frequency(geocat_frequency):
     frequency_mapping = {
-        'continual': 'http://purl.org/cld/freq/continuous',
-        'daily': 'http://purl.org/cld/freq/daily',
-        'weekly': 'http://purl.org/cld/freq/weekly',
-        'fortnightly': 'http://purl.org/cld/freq/biweekly',
-        'monthly': 'http://purl.org/cld/freq/monthly',
-        'quarterly': 'http://purl.org/cld/freq/quarterly',
-        'biannually': 'http://purl.org/cld/freq/semiannual',
-        'annually': 'http://purl.org/cld/freq/annual',
-        'asNeeded': 'http://purl.org/cld/freq/irregular',
-        'irregular': 'http://purl.org/cld/freq/irregular',
+        'continual':
+            'http://publications.europa.eu/resource/authority/frequency/CONT',
+        'daily':
+            'http://publications.europa.eu/resource/authority/frequency/DAILY',
+        'weekly':
+            'http://publications.europa.eu/resource/authority/frequency/WEEKLY',
+        'fortnightly':
+            'http://publications.europa.eu/resource/authority/frequency/BIWEEKLY',
+        'monthly':
+            'http://publications.europa.eu/resource/authority/frequency/MONTHLY',
+        'quarterly':
+            'http://publications.europa.eu/resource/authority/frequency/QUARTERLY',
+        'biannually':
+            'http://publications.europa.eu/resource/authority/frequency/ANNUAL_2',
```

Comment:
I am not sure the mapping of `ANNUAL_2` to `biennually` is correct. I think `ANNUAL_2` means every other (or every second year. And `ANNUAL_3` would mean after three years).
I think this should be `BIENNIAL` (similar to `biweekly`)

Reply:
wait, this comment is probably wrong, let me double-check this

Reply:
I double-checked, how you mapped it, is correct, sorry for the confusion, I got that wrong. Will resolve this thread :)